docs/userguide/developer/recipes.md

```diff
@@ -0,0 +1,130 @@
+# Recipe Guide
+
+Earth2Studio recipes are reference solutions that focus on solving specific use cases.
+Recipes provide more complex implementations that may require deeper domain knowledge
+and familiarity with the codebase.
+
+## What is a Recipe?
+
+While examples demonstrate how to use features, recipes focus on how to solve specific
+problems.
+Recipes are not intended to be turn-key solutions with every potential option; rather
+reference or boilerplate projects for different common use cases.
+Here's how they differ:
+
+### Examples
+
+- Self-contained [JupyText](https://jupytext.readthedocs.io/en/latest/) format
+- Focus on demonstrating specific features
+- Include graphical output for sphinx gallery
+- Requirements must be part of earth2studio + docs dep group
+- Single GPU, A100 (40G) max resource requirement
+- Run in less than 10 minutes
+
+### Recipes
+
+- Require multiple support files
+- Focus on providing complete solutions
+- Configuration YAMLs/JSONs
+- Pythonic interaction through main.py
+- Requirements provided in pyproject.toml
+- May require more specific hardware or system dependencies
+
+See {ref}`examples_userguide` for more information on creating an example.
```

Comment:
This ref isn't rendering in markdown properly for me

Reply:
Ah yes, It wont in markdown but will with the MySt when the docs get built
https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#reference-roles